doc/source/examples/ex_python_uv.rst

```diff
@@ -0,0 +1,167 @@
+.. _example_python_uv:
+
+.. note::
+
+    Go to the `bottom of this page`_ to download the ZIP file for the uv example.
+
+Running arbitrary Python scripts on HPS
+=======================================
+
+This example shows how arbitrary Python scripts can be run on HPS, by
+using `uv <https://docs.astral.sh/uv/>`__ to generate the required
+environments on the fly.
+
+The example sets up a project that shall plot ``sin(x)`` using numpy and
+matplotlib, and then save the figure to a file.
+
+The main feature enabling uv to take care of the environment setup is
+the metadata header present in the ``eval.py`` script, which defines the
+dependencies:
+
+.. code:: python
+
+   # /// script
+   # requires-python = "==3.12"
+   # dependencies = [
+   #     "numpy",
+   #     "matplotlib"
+   # ]
+   # ///
+
+More information can be found `here
+(python.org) <https://packaging.python.org/en/latest/specifications/inline-script-metadata/#inline-script-metadata>`__
+and `here
+(astral.sh) <https://docs.astral.sh/uv/guides/scripts/#running-a-script-with-dependencies>`__.
```

Comment:
```suggestion
For more information, see `Inline script metadata
<https://packaging.python.org/en/latest/specifications/inline-script-metadata/#inline-script-metadata>`__
in the *Python Packaging User Guide* and `Running a script with dependencies <https://docs.astral.sh/uv/guides/scripts/#running-a-script-with-dependencies>`__
in the uv documentation.
```
You should not use "here" but rather the name of the article or topic that the link takes you to and the name of the guide/publication it is in.